test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -74,4 +74,9 @@ Feature: Status Desktop Chat
 	Examples:
 		| displayName        |
 		| notExistingAccount |
-		| asdfgNoNo          |
\ No newline at end of file
+		| asdfgNoNo          |
+
+    Scenario: User can send an emoji in a message
+         When user joins chat room automation-test
+         Then user is able to send the emoji heart_eyes as a message
```

Comment:
WDYT if we split into action and validation like:
- `When the user sends the emoji`
- `Then the message with emoji is displayed`